src/resources/js/commerce/gateway/base.js

```diff
@@ -25,3 +25,57 @@ tribe.tickets.commerce = tribe.tickets.commerce || {};
  */
 tribe.tickets.commerce.gateway = tribe.tickets.commerce.gateway || {};
 
+/**
+ * Path to the billing information in the global tribe Object
+ *
+ * @since TBD
+ *
+ * @type {Object}
+ */
+tribe.tickets.commerce.billing = {};
+
+/**
+ * Selectors used to store billing information
+ *
+ * @since TBD
+ *
+ * @type {{billingLastName: string, billingEmail: string, cardZip: string, billingFirstName: string}}
+ */
+tribe.tickets.commerce.billing.selectors = {
+	billingFirstName: '#tec-tc-gateway-stripe-billing-first-name > input',
+	billingLastName: '#tec-tc-gateway-stripe-billing-last-name > input',
+	billingEmail: '#tec-tc-gateway-stripe-billing-email > input',
+	cardZip: '#tec-tc-gateway-stripe-card-zip > input',
```

Comment:
Do we have control over these inputs? Normally, in the other JS libraries, we rely on classes and we're more specific. We put a particular class to every input on the form and we use that instead of the `#tec-tc-gateway-stripe-billing-first-name > input` (for example.)